packetbeat/docs/reference/configuration/packetbeat-options.asciidoc

```diff
@@ -832,6 +832,22 @@ periodically afterwards, it scans the process table for
 processes that match the values specified for this option. The match is done against the
 process' command line as read from `/proc/<pid>/cmdline`.
 
+
+[[shutdown-timeout]]
+===== shutdown_timeout
+
+How long Packetbeat waits on shutdown. By default, this option is disabled, and Packetbeat
+does not wait for the publisher to finish sending events before shutting down
```

Comment:
We should add here that packetbeat is just going to wait for `shutdown_timeout` and the closes and will not track if all events were sent previously.